v2/emailpassword/pre-built-ui/setup/frontend.mdx

```diff
@@ -257,9 +257,7 @@ Before we initialize the `supertokens-auth-react` SDK let's see how we will use
 **Architecture**
 
 - The `supertokens-auth-react` SDK is responsible for rendering the login UI, handling authentication flows and session management.
-- We could use this SDK for the entire app but it would inflate the bundle size with UI components and dependencies that most of our app won't use.
-- That's where the `supertokens-web-js` SDK comes in. It contains the functions to handle authentication and session management without any of the pre-built UI. In fact, the `supertokens-web-js` SDK is used internally by the `supertokens-auth-react` SDK.
-- We can now reduce the bundle size and load times for all non-auth-related routes in our app by only initialising the `supertokens-auth-react` SDK for auth-related routes and use the `supertokens-web-js` SDK for session management in the rest of our app.
+- After initializing `supertokens-auth-react`, we can use either the `supertokens-auth-react` or `supertokens-web-js` in the application, since the former also initializes the latter.
```

Comment:
Redo this part of the docs to make it very clear what supertokens-auth-react SDK is for, and that adding react won't increase the bundle size of your app other than the auth routes. Do NOT mention web-js

Reply:
Same thing goes for the vue section